src/Vegetation/PlantHydraulics.jl

```diff
@@ -206,30 +209,35 @@ function flux(
     K_sat2,
 ) where {FT}
 
-    S_l1 = inverse_water_retention_curve(vg_α, vg_n, vg_m, p1, ν, S_s)
-    S_l2 = inverse_water_retention_curve(vg_α, vg_n, vg_m, p2, ν, S_s)
+    S_l1 = inverse_water_retention_curve(vg_α, vg_n, vg_m, ψ1, ν, S_s)
+    S_l2 = inverse_water_retention_curve(vg_α, vg_n, vg_m, ψ2, ν, S_s)
 
-    K1_p1 = hydraulic_conductivity(K_sat1, vg_m, S_l1)
-    K2_p2 = hydraulic_conductivity(K_sat2, vg_m, S_l2)
+    K1_ψ1 = hydraulic_conductivity(K_sat1, vg_m, S_l1)
+    K2_ψ2 = hydraulic_conductivity(K_sat2, vg_m, S_l2)
 
-    flux = -(K1_p1 + K2_p2) / 2 * ((p2 - p1) / (z2 - z1) + 1)
+    flux = -(K1_ψ1 + K2_ψ2) / 2 * ((ψ2 - ψ1) / (z2 - z1) + 1)
+    
+    # Future, for science, add this:
+    # suction_term = -1/(z2 - z1) * (p2 - p1) * (K1_p1 + K2_p2)/2
+    # gravity_term = -(K1_p1 + K2_p2)/2
+    # flux = suction_term + gravity_term
 
-    return flux # units of [m s-1]
+    return flux # (m/s)
```

Comment:
this is an example of a helpful comment to that is OK to keep :) although the doc string should also convey this information.

Reply:
which comment though: units of [m s-1] or (m/s)? I shortened it and made it same format everywhere